apps/frontend-manage/src/pages/index.tsx

```diff
@@ -124,22 +151,66 @@ function Index() {
                     className="w-full h-full mt-4 md:overflow-y-auto md:mx-auto"
                     key="question-list"
                   >
-                    <Button
-                      onClick={() =>
-                        setIsQuestionCreationModalOpen(
-                          !isQuestionCreationModalOpen
-                        )
-                      }
-                      className={{
-                        root: twMerge(
-                          'float-right mb-3 font-bold text-white',
-                          theme.primaryBgDark
-                        ),
-                      }}
-                      data={{ cy: 'create-question' }}
-                    >
-                      FRAGE ERSTELLEN
-                    </Button>
+                    <div className="flex flex-row content-center justify-between w-full h-full mt-4 pl-7">
+                      <div className="relative flex flex-row w-2.5/5 pb-6">
+                        <TextField
```

Comment:
Even though it looks fine in the design-system, the icon passed to the TextField component seems to break something here...

<img width="510" alt="Screenshot 2023-01-27 at 17 10 52" src="https://user-images.githubusercontent.com/80708107/215136930-f6b1983b-261a-41b3-9223-a3f98a5c524c.png">

Reply:
as discussed with @rschlaefli yesterday, I applied a temporary fix for now. We'll have to investigate further later on.  @sjschlapbach Could you check again if it works now on your machine as well:

![image](https://user-images.githubusercontent.com/34344089/216078573-52415859-2d2f-4bbf-a235-8e9f87939f06.png)